packages/editor/src/plugins/text/components/suggestion-item.tsx

```diff
@@ -0,0 +1,74 @@
+import IconFallback from '@editor/editor-ui/assets/plugin-icons/icon-fallback.svg'
+import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
+import React, { forwardRef } from 'react'
+
+import { SuggestionOption } from '../hooks/use-suggestions'
+import { cn } from '@/helper/cn'
+
+interface SuggestionItemProps {
+  option: SuggestionOption
+  selected: boolean
+  onSelectPlugin: (
+    event:
+      | React.MouseEvent<HTMLButtonElement>
+      | React.KeyboardEvent<HTMLButtonElement>,
+    pluginType: string
+  ) => void
+  onFocus: () => void
+  onMouseEnter: () => void
+  onMouseLeave: () => void
+  onBlur: () => void
+  tooltipPosition?: 'right' | 'left'
+}
+
+export const SuggestionItem = forwardRef<
+  HTMLButtonElement,
+  SuggestionItemProps
+>(function SuggestionItem(props, ref) {
+  const {
+    option,
+    selected,
+    onSelectPlugin,
+    onFocus,
+    onMouseEnter,
+    onMouseLeave,
+    onBlur,
+    tooltipPosition,
+  } = props
+
+  const { pluginType, title, icon, description } = option
+
+  return (
+    <button
+      data-qa={`plugin-suggestion-${pluginType}`}
+      data-active={selected}
+      ref={ref}
+      onClick={(e) => onSelectPlugin(e, pluginType)}
+      onKeyDown={(e) => {
+        if (e.key === 'Enter') {
+          onSelectPlugin(e, pluginType)
+        }
+      }}
+      onFocus={onFocus}
+      onBlur={onBlur}
+      onMouseEnter={onMouseEnter}
+      onMouseLeave={onMouseLeave}
+      className={cn(
+        'group serlo-tooltip-trigger flex cursor-auto flex-col items-center rounded-md border border-2 border-transparent pb-0'
+      )}
+    >
+      <EditorTooltip position={tooltipPosition} text={description} />
+      <span
+        className={cn(
+          'w-full cursor-pointer rounded-md p-2 hover:shadow-suggestions',
+          selected && 'shadow-suggestions'
```

Comment:
@GregorZupan is that the intended design for a selected/hovered item (with the shadow + the outline)?
<img width="154" alt="image" src="https://github.com/user-attachments/assets/fccb2235-0281-4e9b-877d-a40963df47b5">

Reply:
only when using the keyboard, on mouse hover the outline should not appear (but removing it is not suuuuper crucial)

Reply:
I can only replicate this issue once i interqcted with my keyboard and **then** use the mouse. Once i click into empty space, this behaviour stops